transmart-api-server/src/main/webapp/WEB-INF/keycloak.json

```diff
@@ -0,0 +1,10 @@
+{
+  "realm": "transmart",
```

Comment:
we don't need this one anymore, right?

Reply:
as I mentioned above, I did not remove all the KeycloakRestTemplate settings, so maybe it is still using it for the GB calls. I will test if it still works after removing this config.